lib/utils/utils.js

```diff
@@ -79,7 +79,7 @@ var utils = module.exports = {
   },
   xmlEncode: function(text) {
     // eslint-disable-next-line no-control-regex
-    return text.replace(/[<>&'"\x7F\x00-\x08\x0A-\x0C\x0E-\x1F]/g, function(c) {
+    return text.replace(/[<>&'"\x7F\x00-\x08\x0B-\x0C\x0E-\x1F]/g, function(c) {
```

Comment:
Could you also add a unit test for this (in spec/unit/utils/utils.spec.js to ensure \x0a is retained